installation/kubernetes.md

```diff
@@ -22,22 +22,108 @@ LangStream container images are available on the [Github packages registry](http
 * [datastax/langstream-api-gateway](https://github.com/LangStream/langstream/pkgs/container/langstream-api-gateway)
 * [datastax/langstream-cli](https://github.com/LangStream/langstream/pkgs/container/langstream-cli)
 
-### Quick Start
+### Quickstart
 
-To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster.
+To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster with a recent version.
 
+Connect to your GCP cluster:
+```bash
+gcloud container clusters get-credentials <langstream-cluster> --region us-east1 --project gcp-techpubs
+Fetching cluster endpoint and auth data.
+Kubernetes v1.21.0-alpha+eabdbc0a40fe7efda92e10270f27b0a3485fb743
+kubeconfig entry generated for langstream-cluster.
+```
+
+Connect to your Azure cluster:
+```bash
+az account set --subscription 249a85c0-95ce-41be-8ede-149bce428ae5
+az aks get-credentials --resource-group k8s-resource-group --name dev
+Merged "dev" as current context in /Users/mendon.kissling/.kube/config
+```
+
+Add the LangStream chart repo to your Helm installation and update it to the latest version:
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/LangStream/langstream/main/helm/examples/minio-dev.yaml
```

Comment:
I would keep this, maybe in a new  section about "minikube" (in case you want to test your application with minikube but without mini-langstream, because maybe you have other components to deploy)